src/infrastructure/ui/pages/home-page/hooks.ts

```diff
@@ -35,7 +48,21 @@ const useHomePageData = () => {
                 return require('~/domain/entities/assets/home-page/home-page-basket-level-11.svg');
         }
     };
-    return { username, chooseRightDynamicImage, anecdote, ecoScore };
+
+    const getRandomNumberInArrayLength = (arrayLength: number): number => {
+        return Math.round(Math.random() * arrayLength);
```

Comment:
Chipotage mais peut-être dans les helper créer une méthode randInt spécialement pour ça
```suggestion
        return randInt(arrayLength);
```

Car on risque d'en avoir besoin à nouveau non ?
Le dossier sera un truc comme ça :
```
function randInt(): number;
function randInt(maxValue: number): number;
function randInt(minValue: number, maxValue: number): number;
function randInt(...args: number[]): number {
    if (args.length == 0) {
        return Math.round(Math.random() * 10);
    }
    if (args[0] < 1) {
        throw new Error('Bad args: maxValue must be upper than 0');
    }
    if (args.length == 1) {
        return Math.round(Math.random() * args[0]);
    } else {
        if (args[1] < 1) {
            throw new Error('Bad args: minValue must be upper than 0');
        } else if (args[0] >= args[1]) {
            throw new Error('Bad args: minValue must be lower than maxValue');
        } else {
            return Math.round(Math.random() * (args[1] - args[0]) + args[0]);
        }
    }
}

export default randInt;
```